c2corg_ui/templates/helpers/view.html

```diff
@@ -74,6 +74,15 @@
   ${get_attr(version, 'comment', md=False)  if version['comment'] else '' | n}
 </%def>
 
+<%def name="show_article_title(locale, is_tab_title=False)">\
```

Comment:
You might indeed have to use ``show_route_title``.
It's probably related to this issue: https://github.com/c2corg/v6_ui/issues/594